bundles/org.openhab.ui/web/src/pages/settings/things/things-list.vue

```diff
@@ -58,7 +58,10 @@
               Alphabetical
             </f7-button>
             <f7-button :active="groupBy === 'binding'" @click="switchGroupOrder('binding')">
-              By binding
+              By Binding
```

Comment:
If you're changing the capitalization then it should be changed as well on every list where there's this type of control (inbox, pages...).

Reply:
not sure what the correct capitalization has to be. As a german, I always have problems with english capitalization-rules.
Most probably I did it just wrong and will let it unchanged.

Reply:
```suggestion
              By binding
```